test/unit/session.spec.js

```diff
@@ -477,17 +528,21 @@ describe('util/session', () => {
         .respondWithSuccess()
         .afterResponse(() => {
           store.commit('hideAlert');
-          clock.runAll();
+          clock.tick(120000);
           store.state.alert.state.should.be.false();
-        });
+        })
+        .finally(cleanup);
     });
   });
 
   describe('local storage changes', () => {
-    beforeEach(mockLogin);
-
-    it('logs out after sessionExpires changes', () =>
-      load('/users')
+    it('logs out after sessionExpires changes', () => {
+      testData.extendedUsers.createPast(1);
+      const cleanup = useSessions(router, store);
+      setData({ session: testData.sessions.createNew() });
+      return mockHttp()
+        .request(() => logIn(router, store, true))
+        .respondWithData(() => testData.extendedUsers.first())
```

Comment:
Now that `App` isn't setting up the event listener, it's possible to test it using `mockHttp()` (similar to the tests related to session expiration).